### ISSUE TITLE
Minify and fingerprint CSS

### ISSUE THREAD

**Issue by nalsai** [Astrothewhiteshadow/astrothewhiteshadow.github.io#12]
If you move your css file to the assets directory and embed it in the head of your webisode like the following, hugo can minify and fingerprint your css for you.

```
{{ $css := resources.Get "css/styles.css" }}{{ if hugo.IsProduction}}{{ $css = $css | minify | fingerprint }}{{ end }}
<link rel="stylesheet" href="{{ $css.RelPermalink }}">
```

Minifying makes the css file as small as possible to reduce the bandwidth cost when loading it.
Fingerprinting gives the file a unique url, so end users browsers always load the up-to-date css even when an old version is still cached.

**Comment by Astrothewhiteshadow** [comment]
minor spelling mistake (
> If you move your css file to the assets directory and embed it in the head of your _**webisode**_ like the following, hugo can minify and fingerprint your css for you.

)

**Comment by Astrothewhiteshadow** [comment]
closed with a8e43227373475adbf018982e575ed58a42425dc
